.infrastructure/helm-chart/todoapp/templates/app/clusterIp.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Chart.Name }}-service
+  namespace: {{ .Values.common.namespace }}
+spec:
+  type: ClusterIP
+  selector:
+    app: {{ .Values.todoapp.labelName }}
+  ports:
+  - protocol: TCP
+    {{- with .Values.service }}
+    port: {{ .port }}
+    targetPort: {{ .targetPort }}
+    {{- end }}
```

Comment:
Ensure that `.Values.service.port` and `.Values.service.targetPort` are defined in your `values.yaml` file to specify the correct ports for the service.